resources/assets/less/bem/forum-post-info.less

```diff
@@ -75,8 +75,9 @@
     }
 
     &--username {
-      width: 100%;
-      
+      overflow-wrap: anywhere;
```

Comment:
I think the plan is to only use `word-break: break-word` for now and then mass replace with this once Safari catches up. It's currently redundant as the `word-break` below does the same thing (and being web, it will probably stay for like another 100 years).